namespace cpp mjollnir.vigridr

enum GameResult {
  WON,
  TIED,
  LOST
}
